mysql-test/suite/innodb/t/attachable_trx.test

```diff
@@ -1,154 +1,3 @@
 --source include/have_debug.inc
```

Comment:
nit: do you still need this? since mysql-test/include/attachable_trx.inc already contains this..

Reply:
It does it test run time effect there, but MTR framework also greps the testcases for it to classify them, i.e. https://github.com/facebook/mysql-5.6/blob/18381192a55a018c0fb8a838659019f2d1f1b3e4/mysql-test/lib/mtr_cases.pm#L1534